test/doseCalc/test_BEDprojection.m

```diff
@@ -0,0 +1,57 @@
+function test_suite = test_BEDprojection
+
+test_functions = localfunctions();
+
+initTestSuite;
+
+function test_BEDprojectionConstructor
+bed = matRad_BEDProjection;
+assertTrue(isobject(bed));
+assertTrue(isa(bed, 'matRad_BEDProjection'));
+
+%test compute single scenario BED projection 
+function test_BED_computeSingleScenario
+dij.physicalDose{1} = 2;
```

Comment:
shouldn't this be a sparse matrix to ensure correct output etc.?
Having only one value might also be to simplistic (although a single, full double is a valid test case), maybe at least 2x2? But I know that the gradients become a bit more annoying then.